site/homepage/content/release-notes.md

```diff
@@ -8,6 +8,10 @@ title = "Release Notes"
 
 * An admin guide for the CoAP adapter has been added.
 
+### API Changes
+
+* The `getRemoteContainer` method in `org.eclipse.hono.client.HonoConnection` has been renamed to `getRemoteContainerId`.
```

Comment:
maybe we also want to mention the changes to `ConnectionFactory`?

Reply:
Yes, I've added that.